StreamChatClient_v3/Controllers/ChannelController.swift

```diff
@@ -48,6 +48,27 @@ extension Client {
                                                           extraData: extraData)
         return .init(channelQuery: .init(channelPayload: payload), client: self)
     }
+
+    /// Creates a new `ChannelController` that will create new channel with members without id. It's great for direct message channels.
+    /// - Note: The number of members should be more than 1.
+    /// - Parameters:
+    ///   - members: Members for the new channel.
+    ///   - type: Type of the new channel.
+    ///   - team: Team for the new channel.
+    ///   - extraData: Extra data for the new channel.
+    /// - Returns: A new instance of `ChannelController`.
+    public func channelController(with members: Set<UserId>,
```

Comment:
This function probably needs to be throwing and should throw an error, is the `members` array is empty.